file-formats/sito/sito-v1.json

```diff
@@ -0,0 +1,893 @@
+              },
+              "language": {
+                "title": "Language Key",
+                "description": "Language Key",
+                "type": "string",
+                "minLength": 2,
+                "maxLength": 2,
+                "pattern": "^[a-z]+$"
+              }
+            },
+            "additionalProperties": false
+          }
+        },
+        "sit2ObjStruc": {
+          "title": "List of SIT2_OBJ_STRUC",
+          "description": "List of SIT2_OBJ_STRUC",
+          "type": "array",
+          "items": {
+            "title": "SIT2_OBJ_STRUC",
+            "description": "SIT2_OBJ_STRUC",
+            "type": "object",
+            "properties": {
+              "sitnobjstrucisreusable": {
+                "title": "Object Reusable",
+                "description": "Reusability Flag",
+                "type": "string",
+                "enum": [
+                  "yes",
+                  "no"
+                ],
+                "enumDescriptions": [
+                  "Yes",
+                  "No"
+                ]
+              },
+              "sitnobjstructype": {
+                "title": "Situation Object Structure Type",
+                "description": "Object Structure Type",
+                "type": "string",
+                "enum": [
+                  "cds",
+                  "inMemory"
+                ],
+                "enumDescriptions": [
+                  "CDS",
+                  "In Memory"
+                ]
+              },
+              "sitnobjstrucscope": {
+                "title": "Source",
+                "description": "Object Source: SAP or Customer",
+                "type": "string",
+                "enum": [
+                  "sap",
+                  "customer"
+                ],
+                "enumDescriptions": [
+                  "SAP",
+                  "Customer"
+                ]
+              },
+              "sitnobjstrucname": {
+                "description": "DD: Name of a structured object (e.g. entity, context)",
+                "type": "string",
+                "maxLength": 30
+              },
+              "sitnobjstrucid": {
+                "title": "Situation Object Structure ID",
+                "description": "Situation Object Structure ID",
+                "type": "string",
+                "maxLength": 32
+              },
+              "sit2ObjStrucT": {
+                "title": "List of SIT2_OBJ_STRUC_T",
+                "description": "List of SIT2_OBJ_STRUC_T",
+                "type": "array",
+                "items": {
+                  "title": "SIT2_OBJ_STRUC_T",
+                  "description": "SIT2_OBJ_STRUC_T",
+                  "type": "object",
+                  "properties": {
+                    "sitnobjstrucdesc": {
+                      "title": "Situation Object Structure Description",
+                      "description": "Situation Object Structure Description",
```

Comment:
We extract these texts from our Data Elements Domains. Our UA experts want them to be camel case. Would it be really required to parse and change the texts for the json serialization?

Reply:
In general, titles shall be title style, descriptions shall be sentence style

However, for names like here, we usually stick to the definition in SAP term. I guess your UAs know whether they are written upper case. ;) 
So in your case it might be a description like "The quick brown fox jumps over the Situation Object Structure Description" because "Situation Object Structure Description" is defined in SAP term this way. 

We typically struggle, because object types like "class", "tables", and so on are defined as lower case in SAP term.